lib/client/weblogin.go

```diff
@@ -335,7 +335,7 @@ func Find(ctx context.Context, proxyAddr string, insecure bool, pool *x509.CertP
 }
 
 // SSHAgentSSOLogin is used by tsh to fetch user credentials using OpenID Connect (OIDC) or SAML.
-func SSHAgentSSOLogin(login SSHLogin) (*auth.SSHLoginResponse, error) {
+func SSHAgentSSOLogin(login SSHLogin, tc *TeleportClient) (*auth.SSHLoginResponse, error) {
```

Comment:
since this func doesn't actually make any teleport requests, pass in just the browser string instead of the entire `TeleportClient`

Reply:
Ok done.